api/server/server.go

```diff
@@ -324,7 +326,7 @@ func getImagesJSON(eng *engine.Engine, version version.Version, w http.ResponseW
 		Filters: r.Form.Get("filters"),
 		// FIXME this parameter could just be a match filter
 		Filter: r.Form.Get("filter"),
-		All:    toBool(r.Form.Get("all")),
+		All:    boolValue(r, "all"),
```

Comment:
It might have been better to split these PRs into two - one for the new bool/int funcs and one for the builder stuff.